install_add_pod/after/Pods/Pods.xcodeproj

```diff
@@ -50,7 +50,6 @@ Targets:
         Build Settings:
           ARCHS: "$(ARCHS_STANDARD_64_BIT)"
           CLANG_ENABLE_OBJC_WEAK: 'NO'
-          CODE_SIGN_IDENTITY: iPhone Developer
```

Comment:
this is fine to be removed from Pods.xcodeproj. We set it in CocoaPods to whichever targets we need.